tests/ordering/tests.py

```diff
@@ -368,3 +368,20 @@ def test_related_ordering_duplicate_table_reference(self):
         r1 = Reference.objects.create(article_id=self.a1.pk)
         r2 = Reference.objects.create(article_id=self.a2.pk)
         self.assertSequenceEqual(Reference.objects.all(), [r2, r1])
+
+    def test_default_order_by_f_expression(self):
+        """
+        Test of using F expressions in default ordering. (#28335)
```

Comment:
Docstrings should state the expected behavior and omit prefixes like "Tests that" since all tests test things. Ticket references should be reserved for obscure issues (not needed here). These guidelines are from [Python coding style](https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/coding-style/#python-style).